float64 long_dis
float64 lat_dis
